cluster/log-dump/log-dump.sh

```diff
@@ -321,6 +381,7 @@ function find_non_logexported_nodes() {
   succeeded_nodes=$(gsutil ls ${gcs_artifacts_dir}/logexported-nodes-registry) || return 1
   echo "Successfully listed marker files for successful nodes"
   NON_LOGEXPORTED_NODES=()
+  # TODO(pjh): check WINDOWS_NODE_NAMES here?
```

Comment:
Create a k8s issue:`Support dumping logs with log exporter for GCE windows nodes`
and reference that issue here.

Reply:
I filed https://github.com/kubernetes/kubernetes/issues/74439, please add comments there if it doesn't capture what you meant for this.